photoprism/Photo.py

```diff
@@ -61,22 +58,21 @@ def get_album_uid_by_name(self, name):
         uid = None
         for d in data:
             if name == d["Title"]:
-                uid = d["UID"]
-        if uid == None:
-            return False
-        else:
-            return uid
+                return d["UID"]
+        
+        return False
```

Comment:
This maybe causes different behaviour if there more albums with the same name (if that is possible)